etcdmgmt/etcd-mgmt.go

```diff
@@ -28,6 +64,19 @@ func StartEtcd() error {
 
 	initialAdvPeerUrls := "http://" + hostname + ":2380"
 
+	// Checking health of etcd cluster. If etcd daemon already running then
```

Comment:
@kshlm   but if i have go with health checkup approach then, suppose we are having 5 node clustersetup and one node went down then health of etcd will not be good, and upon stopping glusterd etcd is continuously running. upon starting glusterd it will check whether health is good or not if health is not good then it will start it again.
@kshlm  second concern is that if we go through health check approach and etcd is already running but health is not good then upon restarting glusterd will create defaunt process of etcd

storing pid of etcd in seperate file seems to be good approach.